src/async/prepare.js

```diff
@@ -80,6 +81,7 @@ export default function prepare(element: any) {
   }
   PrepareContextProvider.childContextTypes = {
     __PREPARE_STATE__: () => {},
+    __IS_PREPARE__: () => {},
```

Comment:
Should this be false?

Reply:
No, this is context types, which should be a prop type function. But rather than use that, a noop function suffices instead of using a real prop type.